crates/nu-command/src/core_commands/version.rs

```diff
@@ -0,0 +1,369 @@
+use indexmap::IndexMap;
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{Example, IntoPipelineData, PipelineData, ShellError, Signature, Span, Value};
+
+pub mod shadow {
+    include!(concat!(env!("OUT_DIR"), "/shadow.rs"));
+}
+
+#[derive(Clone)]
+pub struct Version;
+
+impl Command for Version {
+    fn name(&self) -> &str {
+        "version"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("version")
+    }
+
+    fn usage(&self) -> &str {
+        "Display Nu version."
+    }
+
+    fn run(
+        &self,
+        engine_state: &EngineState,
+        stack: &mut Stack,
+        call: &Call,
+        input: PipelineData,
+    ) -> Result<PipelineData, ShellError> {
+        version(engine_state, stack, call, input)
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "Display Nu version",
+            example: "version",
+            result: None,
+        }]
+    }
+}
+
+pub fn version(
+    engine_state: &EngineState,
+    _stack: &mut Stack,
+    call: &Call,
+    _input: PipelineData,
+) -> Result<PipelineData, ShellError> {
+    let tag = call.head;
+
+    let mut indexmap = IndexMap::with_capacity(4);
+
+    indexmap.insert(
```

Comment:
I'm wondering if this use of `CARGO_PKG_VERSION` is right considering this https://github.com/baoyachi/shadow-rs#4-done-use-shadow. Let's just run with it and see what happens.